inquire/src/prompts/path_select/mod.rs

```diff
@@ -0,0 +1,503 @@
+
+    /// Default value of vim mode, equal to the global default value [config::DEFAULT_PAGE_SIZE]
+    pub const DEFAULT_VIM_MODE: bool = crate::config::DEFAULT_VIM_MODE;
+
+    /// Default value of showing hidden files
+    pub const DEFAULT_SHOW_HIDDEN: bool = false;
+
+    /// Default help message.
+    pub const DEFAULT_HELP_MESSAGE: Option<&'a str> = Some(
+        "↑↓ to move, space to select one, \
+        → to navigate to path, ← to navigate up, \
+        shift+→ to select all, shift+← to clear, \
+        type to filter",
+    );
+
+    /// Default behavior of keeping or cleaning the current filter value.
+    pub const DEFAULT_KEEP_FILTER: bool = true;
+
+    /// Default value of showing symlinks
+    pub const DEFAULT_SHOW_SYMLINKS: bool = false;
+
+    /// Default value of selecting multiple files
+    pub const DEFAULT_SELECT_MULTIPLE: bool = false;
+
+    /// Default visual divider value.
+    pub const DEFAULT_DIVIDER: &'a str = "-----";
+
+    /// Creates a [PathSelect] with the provided message and options, along with default configuration values.
+    pub fn new(message: &'a str, start_path_opt: Option<T>) -> Self {
+        Self {
+            message,
+            start_path_opt,
+            default: None,
+            divider: Self::DEFAULT_DIVIDER,
+            help_message: Self::DEFAULT_HELP_MESSAGE,
+            show_hidden: Self::DEFAULT_SHOW_HIDDEN,
+            show_symlinks: Self::DEFAULT_SHOW_SYMLINKS,
+            select_multiple: Self::DEFAULT_SELECT_MULTIPLE,
+            page_size: Self::DEFAULT_PAGE_SIZE,
+            vim_mode: Self::DEFAULT_VIM_MODE,
+            formatter: Self::DEFAULT_FORMATTER,
+            keep_filter: Self::DEFAULT_KEEP_FILTER,
+            render_config: get_configuration(),
+            selection_mode: Default::default(),
+        }
+    }
+
+    /// Test if a path is hidden file
+    ///
+    /// ### Problems
+    /// This is missing some things described here:
+    /// https://en.wikipedia.org/wiki/Hidden_file_and_hidden_directory
+    /// - android: .nomedia files that tell smartphone apps not to display/include a folder's contets
+    /// - gnome: filenames listed inside a file named ".hidden" in each directory should be hidden
+    /// - macos: files with Invisible attribute are usually hidden in Finder but not in `ls`
+    /// - windows: files with a Hidden file attribute
+    /// - windows: files in folders with a predefined CLSID on the end of their names (Windows Special Folders)
+    ///
+    /// ```
+    /// use inquire::PathSelect;
+    /// use std::path::Path;
+    ///
+    /// assert!(PathSelect::is_path_hidden_file(Path::new("/ra/set/.nut")));
+    /// assert!(!PathSelect::is_path_hidden_file(Path::new("/ra/set/nut")));
+    /// assert!(PathSelect::is_path_hidden_file(Path::new(".maat")));
+    /// assert!(!PathSelect::is_path_hidden_file(Path::new("maat")));
+    ///
+    /// ```
+    pub fn is_path_hidden_file(t: T) -> bool {
```

Comment:
I think this should be inside PathSelectPrompt and not on PathSelect (the prompt builder API), if I got it right this is only used within the impl and not supposed to be used by the user right?

Reply:
Just the function